beacon-chain/cache/committee_ids.go

```diff
@@ -16,27 +17,30 @@ type committeeIDs struct {
 var CommitteeIDs = newCommitteeIDs()
 
 func newCommitteeIDs() *committeeIDs {
-	cache, err := lru.New(8)
+	maxCommitteesPerEpoch := int(params.BeaconConfig().MaxCommitteesPerSlot * params.BeaconConfig().SlotsPerEpoch)
+	cache, err := lru.New(maxCommitteesPerEpoch)
 	if err != nil {
 		panic(err)
 	}
 	return &committeeIDs{cache: cache}
 }
 
-// AddIDs to the cache for attestation committees by epoch.
-func (t *committeeIDs) AddIDs(indices []uint64, epoch uint64) {
+// AddID adds committee ID for subscribing subnet for the attester and/or aggregator of a given slot.
+func (t *committeeIDs) AddID(slot uint64, committeeID uint64) {
```

Comment:
Not sure this is the correct way for this , we restrict this to only 1 epoch. We need an epoch lookahead for our validators.